proto/xudrpc.proto

```diff
@@ -229,10 +236,10 @@ message GetOrdersRequest {
   uint32 max_results = 2 [json_name = "max_results"];
 }
 message GetOrdersResponse {
-  // A list of buy orders sorted by descending price
-  repeated Order buy_orders = 1 [json_name = "buy_orders"];
-  // A list of sell orders sorted by ascending price
-  repeated Order sell_orders = 2 [json_name = "sell_orders"];
+  // A list of the orders of peers
+  Orders peer_orders = 1 [json_name = "peer_orders"];
+  // A list of your orders
```

Comment:
The use of second person "your" here feels out of place. Maybe "A list of orders placed locally" or "A list of orders belonging to this xud instance."